src/test/three-components/TextureUtils-spec.ts

```diff
@@ -45,22 +47,22 @@ suite('TextureUtils', () => {
     test('loads a valid texture from URL', async () => {
       let texture = await textureUtils.load(EQUI_URL);
       texture.dispose();
-      expect(texture.isTexture).to.be.ok;
+      expect((texture as any).isTexture).to.be.ok;
       expect(textureMatchesMeta(
                  texture, {mapping: 'Equirectangular', url: EQUI_URL}))
           .to.be.ok;
     });
     test('loads a valid HDR texture from URL', async () => {
       let texture = await textureUtils.load(HDR_EQUI_URL);
       texture.dispose();
-      expect(texture.isTexture).to.be.ok;
+      expect((texture as any).isTexture).to.be.ok;
```

Comment:
why is "as any" so often needed? Does that mean TS can't deduce its type?

Reply:
Yes. Unfortunately, Three.js type definitions aren't always complete. `as any` is an escape hatch that basically tells `tsc` it could be anything, so don't assert anything about the type. The less we use it the better. With Three.js we have to use it a fair amount (for now).